tsconfig.json

```diff
@@ -0,0 +1,30 @@
+
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "react-native-gesture-handler": ["./react-native-gesture-handler.d.ts"],
+      "react-native-gesture-handler/Swipeable": ["./Swipeable.d.ts"],
+      "react-native-gesture-handler/DrawerLayout": ["./DrawerLayout.d.ts"],
+    },
+    "esModuleInterop": true,
+    "jsx": "react-native",
+    "lib": ["esnext"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "esnext",
+    "allowJs": true,
+    "declaration": true,
+    "declarationMap": true,
+    "noStrictGenericChecks": false,
+    "forceConsistentCasingInFileNames": true,
+    "noImplicitUseStrict": false,
+
+    "outDir": "./dist"
+  },
+  "exclude": ["node_modules"],
+  "include": ["**/*.ts", "**/*.tsx"]
```

Comment:
It shouldn't include .js files as well?

Reply:
I will check.

Reply:
BTW, `"exclude": ["node_modules"],` is unnecessary, it's [ignored by default](https://www.typescriptlang.org/tsconfig#exclude).

Reply:
Looks like `outDir` too, so we can remove `dist`. We can remove the whole `exclude` then.

Reply:
Must've missed that, thanks.